spec/features/idv/in_person_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe 'In Person Proofing' do
+  include DocAuthHelper
+  include IdvHelper
+
+  it 'works for a happy path', js: true, allow_browser_log: true do
+    sign_in_and_2fa_user
+
+    # welcome step
+    visit idv_doc_auth_welcome_step # only thing used from DocAuthHelper
+    click_idv_continue
+
+    # information step
+    find('label', text: t('doc_auth.instructions.consent', app_name: APP_NAME)).click
+    click_idv_continue
+
+    # upload documents step
+    click_on t('doc_auth.info.upload_computer_link')
+    attach_images_that_fail
+    click_submit_default
+    expect(page).to have_content(t('idv.troubleshooting.options.verify_in_person'), wait: 60)
+
+    # start the IPP flow
+    # WILLFIX: temporarily block the "Leave site?" pop-up from appearing; this can be removed
+    #          when LG-6406 is resolved.
+    execute_script 'window.onbeforeunload = undefined;'
+    click_link t('idv.troubleshooting.options.verify_in_person')
+
+    # location page
+    expect(page).to have_content(t('in_person_proofing.headings.location'))
+    click_idv_continue
+
+    # welcome page
+    expect(page).to have_content(t('in_person_proofing.headings.welcome'))
+    click_idv_continue
+
+    # address page
+    expect(page).to have_content(t('in_person_proofing.headings.address'))
+    click_idv_continue
+
+    # state ID page
+    expect(page).to have_content(t('in_person_proofing.headings.state_id'))
+    click_idv_continue
+
+    # ssn page
+    expect(page).to have_content(t('in_person_proofing.headings.ssn'))
+    click_idv_continue
+
+    # verify page
+    expect(page).to have_content(t('in_person_proofing.headings.verify'))
+    click_idv_continue
+
+    # phone page
+    expect(page).to have_content(t('in_person_proofing.headings.phone'))
+    click_idv_continue
+
+    # password confirm page
+    expect(page).to have_content(t('in_person_proofing.headings.password_confirm', app_name: APP_NAME))
```

Comment:
line got too long 😭 😎 
```suggestion
    expect(page).to have_content(
      t('in_person_proofing.headings.password_confirm', app_name: APP_NAME),
    )
```